streams-tests/jvm/src/test/scala/zio/stream/StreamSpec.scala

```diff
@@ -1772,6 +1773,15 @@ class ZStreamSpec(implicit ee: org.specs2.concurrent.ExecutionEnv) extends TestR
       .map(_ must_=== Left("Ouch"))
   }
 
+  private def zipWithLatest = unsafeRun {
```

Comment:
Oh, could you rewrite this with a latch on one of the streams so it won't be flaky in the CI? Or alternatively with the new MockClock.

Reply:
Oh yeah, good catch! I'm on it.